src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball returnRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));

```

Comment:
make color supplier and random the class level variable 

it should be instantiated once (not every method invocation)
```suggestion
    public Ball returnRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(CONSTANT));

```